### DISCUSSION TITLE
Run as normal user

### DISCUSSION THREAD

**Discussion by ispmarin** [rsadsb/dump1090_rs#104] [Q&A]
Hi all, the only way to dump1090_rs detect the SDR is running as root. Is there a way to run as a normal user?

**Comment by wcampbell0x2a** [comment]
This project uses SoapySDR as a library for reading from a sdr, see if you can run https://manpages.debian.org/testing/soapysdr-tools/SoapySDRUtil.1.en.html ` without root?
